helm/cluster-api-ipam-provider-in-cluster/templates/pools/crs-netpol.yaml

```diff
@@ -45,7 +45,7 @@ metadata:
 spec:
   endpointSelector:
     matchLabels:
-      app: create-in-cluster-ip-pool
+      app: create-in-cluster-ip-pools
```

Comment:
see https://github.com/giantswarm/cluster-api-ipam-provider-in-cluster-app/blob/main/helm/cluster-api-ipam-provider-in-cluster/templates/pools/job.yaml#L10